nailgun/entities.py

```diff
@@ -389,6 +389,8 @@ def read(self, entity=None, attrs=None, ignore=('account_password',)):
         """
         if attrs is None:
             attrs = self.update_json([])
+        if ignore is None:
+            ignore = {'account_password'}
```

Comment:
if we were to pass in ```ignore={'something_else'}```, then would we no longer be ignoring ```'account_password'```?
If so, then that would not be desirable. In that case, would something like this be better?
```
if ignore is None:
    ignore = set()
ignore.add('account_password')
```